docs/cgroup-v2.md

```diff
@@ -3,7 +3,15 @@
 runc fully supports cgroup v2 (unified mode) since v1.0.0-rc93.
 
 To use cgroup v2, you might need to change the configuration of the host init system.
-Fedora (>= 31) uses cgroup v2 by default and no extra configuration is required.
+The following distributions are known to use cgroup v2 by default:
```

Comment:
(I removed "no extra configuration is required" because rootless containers still often need configuration for delegating the cpu/cpuset/io controllers)